tests/network/port_forward.go

```diff
@@ -148,20 +170,27 @@ func createCirrosVMIWithPortsAndBlockUntilReady(virtClient kubecli.KubevirtClien
 
 	vmi, err := virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
 	Expect(err).ToNot(HaveOccurred())
-	vmi = tests.WaitUntilVMIReady(vmi, console.LoginToCirros)
+	vmi = tests.WaitUntilVMIReady(vmi, libnet.WithIPv6(console.LoginToCirros))
 
 	return vmi
 }
 
-func testConnectivityThroughLocalPort(portNumber int) error {
-	return exec.Command("curl", fmt.Sprintf("127.0.0.1:%d", portNumber)).Run()
+func testConnectivityThroughLocalPort(ipFamily k8sv1.IPFamily, portNumber int) error {
+	return exec.Command("curl", fmt.Sprintf("%s:%d", libnet.GetLoopbackAddressForUrl(ipFamily), portNumber)).Run()
 }
 
-func waitForPortForwardCmd(stdout io.ReadCloser, src, dst int) {
+func waitForPortForwardCmd(ipFamily k8sv1.IPFamily, stdout io.ReadCloser, src, dst int) {
 	Eventually(func() string {
 		tmp := make([]byte, 1024)
 		_, err := stdout.Read(tmp)
 		Expect(err).NotTo(HaveOccurred())
 		return string(tmp)
-	}, 30*time.Second, 1*time.Second).Should(ContainSubstring(fmt.Sprintf("Forwarding from 127.0.0.1:%d -> %d", src, dst)))
+	}, 30*time.Second, 1*time.Second).Should(ContainSubstring(fmt.Sprintf("Forwarding from %s:%d -> %d", libnet.GetLoopbackAddressForUrl(ipFamily), src, dst)))
+}
+
+func getMasqueradeInternalAddress(ipFamily k8sv1.IPFamily) string {
+	if ipFamily == k8sv1.IPv4Protocol {
+		return "10.0.2.2"
+	}
+	return "fd10:0:2::2"
```

Comment:
What about using `To4`  here ? 

```golang
ipv6address = "fd10:0::2"
if ipFamily == k8sv1.IPv4Protocol {
    return net.ParseIP(ip6address).To4().String
}
return ipv6address
```

Reply:
[To4](https://pkg.go.dev/net#IP.To4) converts the IPv4 address ip to a 4-byte representation. If ip is not an IPv4 address, To4 returns nil.